include/highfive/bits/H5Node_traits.hpp

```diff
@@ -114,13 +117,32 @@ class NodeTraits {
     /// false
     bool exist(const std::string& node_name) const;
 
+    /// \brief Returns the kind of link of the given name (soft, hard...)
+    LinkType getLinkType(const std::string& node_name) const;
+
+    /// \brief A shorthand to get the kind of object pointed to (group, dataset, type...)
+    inline ObjectType getObjectType(const std::string& node_name) const;
+
+
   private:
     typedef Derivate derivate_type;
 
     // A wrapper over the low-level H5Lexist
     bool _exist(const std::string& node_name) const;
+
+    // Opens an arbitrary object to obtain info
```

Comment:
idem

Reply:
Here it was not documentation, just a developer comment (dual-slash)